app/src/main/java/org/oppia/app/player/audio/LanguageDialogFragment.kt

```diff
@@ -6,69 +6,47 @@ import android.os.Bundle
 import androidx.appcompat.app.AlertDialog
 import androidx.fragment.app.DialogFragment
 import org.oppia.app.R
-import java.util.ArrayList
-
-private const val KEY_LANGUAGE_LIST = "LANGUAGE_LIST"
-private const val KEY_CURRENT_LANGUAGE = "CURRENT_LANGUAGE"
 
 /**
- * DialogFragment that controls language selection in audio and written translations
+ * DialogFragment that controls language selection in audio and written translations.
  */
 class LanguageDialogFragment : DialogFragment() {
   companion object {
-    lateinit var languageInterface1: LanguageInterface
+    lateinit var languageInterface: LanguageInterface
+    lateinit var languageList: List<String>
+    var selectedIndex: Int = 0
     /**
-     * This function is responsible for displaying content in DialogFragment
+     * This function is responsible for displaying content in DialogFragment.
      * @param languageInterface: [LanguageInterface] to send data back to parent
```

Comment:
Nit: please add spacing between the ``@param`` line and the summary fragment above it.